### PR TITLE
Modify API X509_STORE_CTX_get_current_cert() in document text

### PR DESCRIPTION
**Pull request by NorthHill-16**
Modify the documentation text of X509_STORE_CTX_get_current_cert(), the previous text is misleading and vague.

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by t8m — APPROVED** [review]
OK with CLA: trivial.

@vdukhovni Are you OK with the improved text?

File: doc/man3/X509_STORE_CTX_get_error.pod [hunk 1]
```diff
@@ -49,8 +49,10 @@ X509_STORE_CTX_set_error_depth() sets the error I<depth>.
 This can be used in combination with X509_STORE_CTX_set_error() to set the
 depth at which an error condition was detected.
 
-X509_STORE_CTX_get_current_cert() returns the certificate in I<ctx> which
-caused the error or NULL if no certificate is relevant.
+X509_STORE_CTX_get_current_cert() returns the current certificate in
+I<ctx>. If error occurred, the current certificate will be the one
```
Comment (vdukhovni) [inline doc/man3/X509_STORE_CTX_get_error.pod:53]:
```suggestion
I<ctx>. If an error occurred, the current certificate will be the one
```

File: doc/man3/X509_STORE_CTX_get_error.pod [hunk 2]
```diff
@@ -49,8 +49,10 @@ X509_STORE_CTX_set_error_depth() sets the error I<depth>.
...
+that is most closely related to error, or possibly NULL if no such
```
Comment (vdukhovni) [inline doc/man3/X509_STORE_CTX_get_error.pod:54]:
```suggestion
that is most closely related to the error, or possibly NULL if no such
```

**Review by vdukhovni — CHANGES_REQUESTED** [review]

**Comment by NorthHill-16** [comment]
Changes are pushed

**Review by vdukhovni — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master, 3.1, and 3.0 branches. Thank you for your contribution.

**Pull request closed without merging**